rustls/tests/api.rs

```diff
@@ -42,6 +43,7 @@ use rustls::{
 use super::*;
 
 mod common;
+// no need to import `common::Arc` - included by wildcard import below
```

Comment:
This feels like something that was useful to call out to reviewers but isn't likely to be useful to a reader of the code in the future. WDYT about deleting this comment addition?